tslint.json

```diff
@@ -29,7 +29,6 @@
         ],
         "no-arg": true,
         "no-bitwise": true,
-        "no-console": [true, "debug", "info", "time", "timeEnd", "trace"],
```

Comment:
I will console log where I want thank you very much

Reply:
:joy: 
first idam telling you your work hours and now this, smh